frappe/auth.py

```diff
@@ -713,3 +716,8 @@ def validate_api_key_secret(api_key, api_secret, frappe_authorization_source=Non
 def validate_auth_via_hooks():
 	for auth_hook in frappe.get_hooks("auth_hooks", []):
 		frappe.get_attr(auth_hook)()
+
+
+def check_request_ip():
+	if frappe.local.request_ip is None:
+		frappe.throw(title="Error", msg="Request IP is not set")
```

Comment:
I just realized this can potentially break CLI. Let's just set it to `127.0.0.1` if it's not set? 

That is the most common case and localhost makes sense for CLI.

Reply:
This would break all cases of `bench browse`.

(currently only broken when someone has set some value for a user and we try to use bench browse with that user)

Reply:
yep